### ISSUE TITLE
Indentation of code

### ISSUE THREAD

**Issue by PrarabdhGarg** [dvmappdassignments/git-github-and-kotlin-freshers2020#9]
@Ayush-Suman 
Your code is not properly indented. It makes it pretty hard to read, especially when not reading in an editor. 
Please make sure to indent your code

**Comment by ayush-suman** [comment]
Yes, I will keep that in mind. I will add proper indentation before Saturday meet.
